code/game/machinery/shieldgen.dm

```diff
@@ -419,8 +419,8 @@
 		needs_power = TRUE
 		setDir(get_dir(gen_primary, gen_secondary))
 	for(var/mob/living/L in get_turf(src))
-		visible_message("<span class='danger'>\The [src] is suddenly occupying the same space as \the [L]!</span>")
-		L.gib()
+		visible_message("<span class='danger'>\The [src] zaps through [L]!</span>")
```

Comment:
I still think this doesn't really make sense.
The burnard silkwind zaps through the shield wall?

Reply:
The text macro makes this fine.

Reply:
the verb just doesn't make sense